security/cmd/istio_ca/main.go

```diff
@@ -194,11 +196,10 @@ func init() {
 	flags.BoolVar(&opts.selfSignedCA, "self-signed-ca", false,
 		"Indicates whether to use auto-generated self-signed CA certificate. "+
 			"When set to true, the '--signing-cert' and '--signing-key' options are ignored.")
-	flags.StringVar(&opts.selfSignedCAOrg, "self-signed-ca-org", "k8s.cluster.local",
-		fmt.Sprintf("The issuer organization used in self-signed CA certificate (default to %s)",
-			selfSignedCAOrgDefault))
```

Comment:
why can't we use the org and other things in x509? We can set it to some defaults, allow users to override.. IT doesn't hurt does it? Will come in handy when we want to do x509 based stuff..

Reply:
this is per request from @myidpt. The Org was a placeholder and can be set to identityDomain for now. We can always revive it when there is explicit need.

Reply:
@rshriram Generally we don't recommend users to set X.509 cert fields other than the SAN URI field. All information in the cert must be valid and putting unnecessary information in the cert can make the cert issuance hard to maintain in practice.

Reply:
In this particular case, we set Org field because RFC5280 says "The issuer field MUST contain a non-empty distinguished name (DN)", so we use Org field as the place holder, which is the same as the identity domain. As Etai said, we can revive it when there is explicit need.